book.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Book
+  attr_reader :borrowings
```

Comment:
- [ ] Setters for `@borrowings` should be present in the Book class since `@borrowings` is accessed in the Rental class.